find_hero_test.py

```diff
@@ -0,0 +1,19 @@
+import opendota
+
+herolist = opendota.get_hero_list()
+
+i = 1
+
+def find_hero(heroname): 
+  i=1
+  for hero in herolist: 
+    i+=1
+    if (herolist[i]['localized_name'] == heroname):
+      return(herolist[i])
```

Comment:
this looks very similar to other find_hero func. do we need it?

Reply:
nop